.gitignore

```diff
@@ -97,3 +97,4 @@ venv/
 
 # VSCode
 .vscode/
+*.code-workspace
```

Comment:
```suggestion
```
This may be necessary but should be separated into other PRs.

Reply:
Whoops - I ignored this file locally and didn't realize that it would be added to this commit. Thanks for catching this. 

If you don't think that adding this is too minor, I will open a new PR for it. Not sure how relevant it is to others, though.

Reply:
I am not familiar with VS Code so I don't know how common it is. But PR and discussion are always welcome:)